radlibDemoApp/www/js/index.js

```diff
@@ -69,14 +71,13 @@ function getParsed() {
 // testing
 function testing(){

```

Comment:
Would we be able to move this into a different function with a different name? I know we have to keep this for future reference when we finish integration of adding readers into the UI. Other people's testing() functions are different...